src/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs

```diff
@@ -546,25 +546,51 @@ public static void TestDebuggerAttributes()
         }
 
         [Fact]
-        public static void TestConstructor_Negative()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18432 yet.")]
+        public static void TestNullComparer()
```

Comment:
> I'm not aware of any other comparer source it could even be defaulting to.

In theory it could for example just fall back to using Object.GetHashCode/Equals. It'd be nice if a test could detect that, though I don't think it's possible to be perfect here. Just something a bit more rigorous would be good. Thanks.